### ISSUE TITLE
read down-sampled stack in parallel

### ISSUE THREAD

**Issue by raacampbell** [alexanderbrown/masiv#6]
Can be very slow to load over the network. Should be faster if we parfor the loading.

**Comment by alexanderbrown** [comment]
Problem is that it's a single file so i'm not sure it can be parallelised. I'll take a look.

Not sure why it should be particularly slow over the network though?

**Comment by raacampbell** [comment]
At least with `imread` on a multi-page TIFF I know it's possible to read the individual pages in parallel. It's already done with a `for` loop so simply making it `parfor` should be sufficient.

**Comment by alexanderbrown** [comment]
I'll give it a go

**Comment by alexanderbrown** [comment]
Just had a quick look. It may work for files read using the `imread` approach, but most files need to be read using a `Tiff` object which won't easily work in parallel. I'll try a couple of things to get it to work
